sound/soc/sof/sof-of-dev.c

```diff
@@ -56,7 +56,11 @@ static void sof_of_probe_complete(struct device *dev)
 	/* allow runtime_pm */
 	pm_runtime_set_autosuspend_delay(dev, SND_SOF_SUSPEND_DELAY_MS);
 	pm_runtime_use_autosuspend(dev);
+	pm_runtime_set_active(dev);
```

Comment:
Hmm, it's a bit weird the sequence is different between OF and PCI, but this is probably the right thing to do and sof-pci-dev.c should call set_active() as well.

Reply:
@kv2019i the PCI core already does this, see drivers/pci/pci.c

Reply:
@plbossart Aa, ok, thanks. I that makes sense. I was grepping through drivers, but redoing that, yeah, they are non-PCI devices.